Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -8,7 +8,7 @@
 
 import XCTest
 
-class PaymentSheetUITest: XCTestCase {
+class PaymentSheetUITestCase: XCTestCase {
```

Comment:
seems like we should update the filename and comment at the top to `PaymentSheetUITestCase`